src/lib/components/get-demo/calendly-embed.svelte

```diff
@@ -0,0 +1,20 @@
+<style lang="postcss">
+  div {
+    @media (min-width: 1091px) {
+      @apply -mt-small;
+    }
+  }
+</style>
+
+<svelte:head>
+  <script
+    type="text/javascript"
+    src="https://assets.calendly.com/assets/external/widget.js">
+</script>
```

Comment:
Fix calendly widget javascript src url

```suggestion
  <script
    type="text/javascript"
    src="https://calendly.com/assets/external/widget.js">
  </script>
```